install.js

```diff
@@ -31,17 +31,7 @@ async function downloadFile(url, destination) {
 
 (async () => {
 	if (platform === 'win32') {
-		if (arch === 'x64') {
-			return downloadFile(urls.windows.x64, 'bin/windows/x64');
-		}
-
-		if (arch === 'ia32') {
-			return downloadFile(urls.windows.ia32, 'bin/windows/ia32');
-		}
-
-		if (arch === 'arm64') {
-			return downloadFile(urls.windows.arm64, 'bin/windows/arm64');
-		}
+		if(urls.windows[arch]) return downloadFile(urls.windows[arch], `bin/windows/${arch}`);
```

Comment:
There should be a space between the if keyword and the opening bracket and a curly brace scope should still be created.